src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/CpsPackageReferenceProject.cs

```diff
@@ -116,6 +113,12 @@ private PackageSpec GetPackageSpec()
             return null;
         }
 
+        internal override async ValueTask<PackageSpec> GetPackageSpecAsync(CancellationToken ct)
```

Comment:
Does this one have to be async? It's just wrapping around non-async method. It adds another machine state.

Reply:
Other override in LegacyPackageReferenceProject.cs is also another wrapper around non-async value.

Reply:
Will be addressed in https://github.com/NuGet/Home/issues/11397